Tests/OpenFeatureTests/OpenFeatureClientTests.swift

```diff
@@ -41,6 +41,29 @@ struct OpenFeatureClientTests {
         }
     }
 
+    @Suite("String")
+    struct StringTests {
+        @Test("value", arguments: ["foo", "bar"])
+        func value(_ defaultValue: String) async {
+            let provider = OpenFeatureDefaultValueProvider()
+            let client = OpenFeatureClient(provider: { provider })
+
+            let value = await client.value(for: "flag", defaultingTo: defaultValue)
```

Comment:
I just realized (and this is the case for the Bool tests too) that we probably don't want to default to the same value here as we return from the provider. Otherwise the same test could still run even if the client would never invoke the provider and always returns the default value.

Reply:
Perhaps we could instead use something like an `OpenFeatureStaticProvider`, hard-coded to the test parameter, and use a completely unrelated value for the `defaultValue`. What do you think?